examples/MANN/README.md

```diff
@@ -1,6 +1,6 @@
 # 🤖 MANN
 
-This tutorial aims to give the user a receipt that can be followed to use the `MANNAutoregressive` and `MANNTrajectoryGenerator` classes to generate a trajectory for the `ergoCub` robot.
```

Comment:
I would move the VelMANN tutorial to a separate folder. Even if it is similar to the MANN one

Reply:
Done in dcf45280.